manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/tooling/Document.scala

```diff
@@ -241,13 +241,7 @@ case class GraphViz(s: String) extends Content with NoQueries {
 
 case class ExecutionPlan(planString: String) extends Content with NoQueries {
   override def asciiDoc(level: Int) = {
-    s""".Query plan
-       |[source]
-       |----
-       |$planString
-       |----
-       |
-       |""".stripMargin
+    s".Query plan\n[source]\n----\n$planString\n----\n\n"
```

Comment:
I don't understand this change. I think the old code better shows what the asciidoc will look like.